.github/ISSUE_TEMPLATE/Bug.md

```diff
@@ -0,0 +1,47 @@
+---
+name: 🐞 Bug Report
+about: Something is broken? 🔨
+---
+
+### Bug Report
+
+<!-- Fill in the relevant information below to help triage your issue. -->
+
+|    Q        |   A
+|------------ | ------
+| BC Break    | yes/no
+| Version     | x.y.z
```

Comment:
Can we add a hint here how people can get the version number? For example, if it's the client app, they can find it as an attribute on the `<body>` element.

Reply:
Sure! I thought the version should be the one inserted in client/server/cli/... package.json. Maybe I'm wrong. Where is this <body> element?

Reply:
Yes it's also in the `package.json` of every package inside `machinelabs/machinelabs`. But user may not actually see that. They just use the service and realize there's a bug. 

In fact, I think unless the issue originates from a developer, there's (right now) only `cli` and `client` that can be of interest.